guillotina/traversal.py

```diff
@@ -287,8 +290,11 @@ async def handler(self, request):
         else:
             task = request.execute_futures("failure")
 
-        if self.wait(request):
-            await asyncio.wait_for(task, app_settings.get("max_wait_futures"))
+        wait = self.wait(request)
```

Comment:
@vangheem what do you think :)

Reply:
no way IMO

All the APIs also say "after_request"

Reply:
eh, up to you. Seems a bit crazy IMO.

Reply:
Ok lets wait not for it :)